scrapfly/scrape_config.py

```diff
@@ -41,6 +42,7 @@ class ScrapeConfig:
     wait_for_selector: Optional[str] = None
     session_sticky_proxy:bool = True
     screenshots:Optional[Dict]=None
+    screenshot_flags: Optional[str]=None
```

Comment:
For developer experience, having a List of str is more convenient, and we can leverage https://docs.python.org/3/library/enum.html to reference all values instead of hardcoding them and get autocomplete/type inference. Then during API param construction https://github.com/scrapfly/python-scrapfly/pull/18/files#diff-45789e70641abe5822f8612ab9c56dfccbb6584e6892d6ca77f12f77004975ddR342 reconstruct from the list the correct format `','.join(flags)`, also make sure there are unique using a set. 

The same applies to format values. We can use an enum and type against it for auto-complete

This will also pre-validate the value